packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -154,6 +154,55 @@ async fn create_entity<P: StorePool + Send>(
         .map(Json)
 }
 
+#[derive(Serialize, Deserialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+struct ArchiveEntityRequest {
+    entity_id: EntityId,
```

Comment:
Oh that is true, we do. Oversight from my end, but we would definitely want this here. I think i treat `owned_by_id` differently than `account_id` in my head, but for sure we should pass this down and make it part of the query. Will update

Reply:
Just flagging that when adding this field, it would only be passed down to the `move_entity_to_histories` query itself. If i were to add this to the other queries, I would have to change how updating works, which would require making changes up the stack to the workspace API. Is it good enough for now that I add the field and pass it down to `move_entity_to_histories` only for now?

Reply:
I think _for now_ that's good enough, yes. We'll probably need to revisit the rest of the code as part of.. maybe this task https://app.asana.com/0/1201095311341924/1203214689883091

Reply:
Added in 28468a8f66f27d893bf6330aa93f9d165b674419 with links to the asana task in todos